core/modules/system/system.install

```diff
@@ -3802,6 +3786,26 @@ function system_update_1102() {
   }
 }
 
+/**
+ * Set the default Basis settings for CSS updates.
+ */
+function system_update_1103() {
+  $config = config('basis.settings');
+  $data = $config->getData();
+  // Put the _config_static option at the top of the file.
+  $data = array_merge(array(
+    '_config_static' => TRUE,
```

Comment:
This change seems problematic for several reasons:

1. This special cases Basis - none of the other theme settings will be statically cached.
2. This special cases Basis installed *before* 1.30, as new installs won't be statically cached.
3. This change seems unrelated to the actual problem (passing null to param...) and deserves its own issue and discussion - for all themes, not only basis.

Reply:
Oh, now I realize that much of this was already part of system_update_1100()

Reply:
Thinking this over a bit - special casing already happened in the previous change, so is unrelated here, anyway.

Consider this as resolved - I can't mark the conversation as resolved GH permission-wise. :wink: